cmd/remove-members/main_test.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestReadMemberList(t *testing.T) {
+	memberList, err := readMemberList("test/members.txt")
+	if err != nil {
+		t.Error(err)
+	}
+
+	expectedMemberList := []string{"johndoe", "janedoe"}
+
+	if !reflect.DeepEqual(memberList, expectedMemberList) {
+		t.Errorf("Values mismatch. Expected: %v Actual: %v", expectedMemberList, memberList)
+	}
+}
+
+func TestRemoveMembers(t *testing.T) {
+
+	memberList := []string{"johndoe", "janedoe"}
+
+	if err := removeMembers(memberList, "test"); err != nil {
```

Comment:
So now that I've gotten here, stepping back a bit, it would be cool/ideal if this was all structured in such a way that you could write tests that:
- don't interact with the filesystem
- don't call git

The filesystem thing might be more effort than it's worth (mock out walk, pass in a tree).  But the git thing... maybe write a git interface with `add` and `commit` methods, and pass in a mock in these tests?

https://github.com/kubernetes/test-infra/blob/master/prow/repoowners/repoowners_test.go would be an example of unit tests that dealt with files in a tree layout